src/loaders/it_load.c

```diff
@@ -963,6 +963,11 @@ static int load_it_pattern(struct module_data *m, int i, int new_fx,
 		return -1;
 	}
 
+	/* fill event bases for this pattern */
+	for (c = 0; c < L_CHANNELS && c < m->mod.chn; c++) {
+		event_base[c] = m->mod.xxt[TRACK_NUM((i),(c))]->event;
+	}
+
```

Comment:
This looks fragile and non-futureproof. I recommend keeping `EVENT()` as-is.

Reply:
Actually, a followup here: do you see similar performance improvements by changing the function declaration to this?
```C
static int load_it_pattern(struct module_data * LIBXMP_RESTRICT m, int i, int new_fx,
			   HIO_HANDLE * LIBXMP_RESTRICT f)
```